onyo/commands/tests/test_mkdir.py

```diff
@@ -132,17 +132,17 @@ def test_mkdir_quiet_flag(repo: OnyoRepo) -> None:
 
 @pytest.mark.repo_dirs(*directories)
 @pytest.mark.parametrize('directory', directories)
-def test_error_dir_exists(repo: OnyoRepo, directory: str) -> None:
+def test_dir_exists(repo: OnyoRepo, directory: str) -> None:
     """
     Test the correct error behavior when `onyo mkdir <path>` is called on an
     existing directory name.
     """
     ret = subprocess.run(['onyo', 'mkdir', directory], capture_output=True, text=True)
 
     # verify output
-    assert not ret.stdout
-    assert "The following paths already exist:" in ret.stderr
-    assert ret.returncode == 1
+    assert "No assets updated." in ret.stdout
```

Comment:
"assets" seems to be the wrong term IMO. Either directories, or even `.anchor` files are reported with `onyo mkdir`. Especially once asset dirs exist this would became missleading.

Reply:
Yes. That's a separate issue, though: #374